src/main/scala/scalismo/package.scala

```diff
@@ -38,6 +38,10 @@ package object scalismo {
       gc.SetScheduleTime(60, TimeUnit.SECONDS)
       gc.Start()
 
+      // we need to stop the garbage collection when the JVM shutdowns.
+      // Otherwise it prevents the JVM from shutting down correctly.
+      Runtime.getRuntime().addShutdownHook(new Thread { gc.Stop() })
```

Comment:
Yeah, I thinks that's probably a sensible and working solution. A few comments though:
- VTK is extremely sensitive to threading issues, so we should mimic the "original" behavior as much as possible. That means running the actual GC in the Swing EDT (runLater(), cf. the source code of the VTK-provided auto-GC implementation).
- should we allow for customizations of the interval and/or debug flag, or is this "YAGNI" territory :-) ?
- what about the (unusual and improbable) case when someone/something else explicitly starts the original VTK auto-GC?